lib/internal/streams/utils.js

```diff
@@ -27,7 +30,16 @@ function isIterable(obj, isAsync) {
     typeof obj[SymbolIterator] === 'function';
 }
 
+function isDisturbed(stream) {
+  return !!(stream && (
+    stream.readableDidRead ||
+    stream.readableAborted ||
+    stream[kIsDisturbed]
+  ));
+}
+
 module.exports = {
+  isDisturbed,
```

Comment:
```suggestion
  isDisturbed,
  kIsDisturbed,
```

Not sure how the tests pass without this, but `kIsDisturbed` is required in `lib/internal/webstreams/readablestream.js`